jwst/scripts/set_telescope_pointing.py

```diff
@@ -163,8 +189,23 @@
                 method=args.method,
                 j2fgs_transpose=args.transpose_j2fgs,
                 save_transforms=transform_path,
-                override_transforms=override_transforms
+                override_transforms=override_transforms,
             )
         except (TypeError, ValueError) as exception:
             logger.warning('Cannot determine pointing information: %s', str(exception))
             logger.debug('Full exception:', exc_info=exception)
+
+
+def deprecated_name():
+    from pathlib import Path
+
+    filename = Path(__file__)
+    warnings.warn(
+        f'usage of `{filename.name}` is deprecated; use `{filename.stem}` instead'
+    )
+
+    main()
```

Comment:
added this function to warn for use of deprecated entry point

Reply:
This should work fine. Thanks!